src/api/commonApi.ts

```diff
@@ -1,13 +1,22 @@
-import { authenticatedApiRequest2 } from './apiRequest';
+import { apiRequest2, authenticatedApiRequest2 } from './apiRequest';
 
-export const getById = async <T>(id: string) => {
+export const getByIdAuthenticated = async <T>(id: string) => {
```

Comment:
Usikker på naming for `getById` og `getByIdAuthenticated` her.
Er `fetchResource` og `fetchProtectedResource` e.l. bedre?

Reply:
Liker `fetchResource` og `fetchProtectedResource` bedre. Overflødig med 'ById' da det er parameteret og man spesifiserer type i generic. Også enig i at det å separere med og uten token er ryddigere.

Reply:
da endrer jeg til det